Resources/Prototypes/Floof/Entities/Objects/Specific/Robotics/Borgmodules.yml

```diff
@@ -36,4 +36,35 @@
     items:
     - WeaponAdvancedLaser
     - WeaponborgPistolMk58
-    - CombatKnife
\ No newline at end of file
+    - CombatKnife
+    
+- type: entity
+  id: BorgModuleInvestigation
+  parent: [ BaseBorgModuleSecurity, BaseProviderBorgModule ]
+  name: Investigation cyborg module
+  components:
+  - type: Sprite
+    sprite: Floof/Objects/Specific/Robotics/borgmodule.rsi
+    layers:
+    - state: security
+    - state: icon-investigation
+  - type: ItemBorgModule
+    items:
+    - ForensicScanner
+    - DetectivePDA
```

Comment:
Giving a full PDA seem just... bad, there should be another way.

Reply:
to my knowledge there is not to be able to allow them to utilize logprobing with doors